KMS.md

```diff
@@ -152,6 +152,38 @@ The URI format for Hashicorp Vault KMS is:
 This provider requires that the standard Vault environment variables (VAULT_ADDR, VAULT_TOKEN) are set correctly.
 This provider also requires that the `transit` secret engine is enabled
 
+If you enabled `transit` secret engine at different path with the use of `-path` flag (i.e., `$ vault secrets enable -path="someotherpath" transit`), you can use `TRANSIT_SECRET_ENGINE_PATH` environment variable to specify this path while generating a key pair like the following:
```

Comment:
IIRC, @dlorenc would want to move documentation parts from cosign repository to the sigstore website which you can find [here](https://github.com/sigstore/sigstore-website), so, we added all the details related to Vault here, please do check it out first, then suggest your updates here.

👉 https://docs.sigstore.dev/cosign/kms_support#hashicorp-vault